### ISSUE TITLE
Per-folder duplication?

### ISSUE THREAD

**Issue by whoiswes** [trapexit/mergerfs#173]
TrapExit,

I realize this is probably far outside the purview of mergerFS, but figured I'd ask.  I've recently moved from AUFS to mergerFS for my home server, and am very pleased with how well it's working.  One thing that has always been on my wishlist for the various drive pooling solutions is folder duplication.  For instance, if i have /mnt/disk1 and /mnt/disk2 in a pool, it would be beneficial for me to be able to duplicate the pictures top-level folder across both disks, while leaving every other folder as a single copy.

In other words, Greyhole without the overhead or StableBit DrivePool without Windows.

Again, I realize this is probably well outside of what mergerFS can/should do, but wanted to mention it regardless in case it sparked any light bulbs for you.

Many thanks!

Wes

**Comment by trapexit** [comment]
Duplicating data inline is a difficult thing to do safely and for something like that best left out of band. A simple rsync + cron would probably serve you fine and be far simpler/safer.

Doing something like that in mergerfs or any FS layer means you have to write data multiple times (simple) and deal with errors gracefully (not simple).
